docs: Add information about available packages

CarwynNelson opened the pull request:
There are community maintained packages for Arch and Nix which some may prefer to use. Highlight these in the documentation.

Resolves #1

Commit:
docs: Add information about available packages

There are community maintained packages for Arch and Nix which some may
prefer to use. Highlight these in the documentation.

jonhoo commented on README.md:
```diff
@@ -1,13 +1,46 @@
 
-and then use the Yambar config further down in the README. You'll need
-to adjust the `script` path.
+Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
```
```suggestion
Use `~/.cargo/bin/yambar-hyprland-wses` as your script path
```

jonhoo replied:
I recently discovered that yambar specifically supports paths that start with `~/` (ref yambar-modules-script(5))

CarwynNelson replied:
Nice to know that exists; I have updated the docs to reflect that.

jonhoo commented on README.md:
```diff
@@ -1,13 +1,46 @@
+## Arch Linux
+
+There is a community maintained package available in the AUR. Assuming you have
+[yay], run:
```
We could also link to [paru](https://github.com/Morganamilo/paru) which is also written in Rust (like this is!). It's what I personally use. Maybe insert a generic mention here of "any AUR helper". Could also give the generic AUR install instructions:

```console
$ git clone https://aur.archlinux.org/yambar-hyprland-wses.git
$ cd yambar-hyprland-wses
$ makepkg -si
```

CarwynNelson replied:
Ah yes it does feel right for a Rust utility to mention a Rust AUR helper :)

I have added instructions for Paru and manual installation

jonhoo commented on README.md:
```diff
@@ -1,13 +1,46 @@
+Yambar does not check the `$PATH` for scripts which means you cannot just use
+`yambar-hyprland-wses` as the script path. It is probably easiest for Nix users
+to generate the yambar config in your nix config, which will know the correct
+path to `yambar-hyprland-wses`.
```
Maybe include a line for how they get that path if they nixify their config?

CarwynNelson replied:
I did a bit of digging and found out how you generate that path in nix config.

I must admit I have barely used Nix so I'm no expert on it. I'm wondering if its worth either:
1. Removing the section on the Nix package as it is mostly a guess, or
2. Simplifying the section to note that there is a nix package if people want it, but you are on your own a little bit with configuration, and PRs for better docs on the Nix package are welcome

Do you have any thoughts?

jonhoo replied:
Let's go with option 2 :+1:

jonhoo reviewed (changes requested):
Nice! Mostly nits :)

CarwynNelson commented:
> Nice! Mostly nits :)

Nits welcome :)

Commit:
docs: Use relative path for cargo script

This will make the documentation easier to copy/paste for users
installing the package via `cargo install`

See: yambar-modules-script(5)

Commit:
docs: Expand AUR installation instructions

- Include Paru instructions
- Include manual installation instructions

CarwynNelson commented:
I have updated the PR with the requested changes. Thanks for taking the time to review my pr I really appreciate it :)

jonhoo commented on README.md:
````diff
@@ -13,7 +13,7 @@ Assuming you have [Rust installed][Rust], run:
 ```
 
-Use `/home/yourusernamehere/.cargo/bin/yambar-hyprland-wses` as your script path
+Use `/.cargo/bin/yambar-hyprland-wses` as your script path
````
Ah, this isn't quite right. You need the `~` at the beginning of the path to indicate the home directory, and the same in the YAML down below.

CarwynNelson replied:
Wow, I swear I added `~` at the front :sweat_smile:

I'll get that sorted.

jonhoo reviewed (approved):
Looks great, thank you!
